### ISSUE TITLE
issues with node refinement between electrodes using paraDX

### ISSUE THREAD

**Issue by benbpete** [gimli-org/gimli#430]
### Problem description
When creating a PLC and using paraDX, values less than 0.33, eg 0.25 or 0.1, do not result in equally spaced nodes between electrodes.

### Your environment

Windows 10
Python version: 3.8.13
pygimli version: 1.3.0
Conda install

### Steps to reproduce

When creating a PLC, such as below:

```
import pygimli as pg
geo = pg.meshtools.createParaMeshPLC(data, paraDX = .25)
...
```

### Expected behavior

a PLC, based on the data set, will be created with node refinement between the electrodes of 0.25.

### Actual behavior
Node refinement occurs adjacent to the electrodes. Additional nodes between the nodes adjacent the electrodes are not incorporated. See attached .png examples.  The suffix _5, _25, etc describes the paraDX value used to create the PLC.  Thank you!

Ben

![T5_robust_bndry1_PLC_lim_5](https://user-images.githubusercontent.com/105517351/188729299-b4f62a6b-b2e2-4562-aab9-97a868554c8e.png)
![T5_robust_bndry1_PLC_lim_25](https://user-images.githubusercontent.com/105517351/188729301-6a00b8bf-c1ed-4d3e-a676-68beee1f16ac.png)
![T5_robust_bndry1_PLC_lim_33](https://user-images.githubusercontent.com/105517351/188729303-c697de38-3935-41dc-aa43-ffd1616b72b4.png)
![T5_robust_bndry1_PLC_lim_1](https://user-images.githubusercontent.com/105517351/188729304-05644966-895f-4f8f-937d-92801e68b5d4.png)

**Comment by halbmy** [comment]
I see the point. With a sufficiently high triangle quality, the node spacing at the surface will become quasi-equistant, but it would be good to have this as option (as we had in the BERT cfg file). There is already an option `addNodes` specifying the number of equidistant nodes between sensors but it is currently only used for closed domains (`isClosed=True`). We could easily use this also for surface meshes. Thank you for pointing this out.

**Comment by halbmy** [comment]
I quickly implemented this by e5b4a48:
The default behaviour stays the same, i.e. paraDX will just locally insert nodes (like in 3D). If `addNodes>1` given (the default `addNodes=1` equals the default `paraDX=0.5`), equidistant nodes are created. Example for `addNodes=5`:

![image](https://user-images.githubusercontent.com/7136401/188806794-b3c720fb-6154-4368-877a-c1696d241d67.png)

**Comment by benbpete** [comment]
This is great. Thank you for making the changes!
